build-tools/automation/yaml-templates/install-signing-plugin.yaml

```diff
@@ -0,0 +1,57 @@
+parameters:
```

Comment:
Is it worth putting `-plugin` in the file name? I know they named it `MicroBuildSigningPlugin`, but it seems like they picked a weird name for a `.yaml` template.

Reply:
I'll update these names to `[install][remove]-microbuild-tooling`, which will hopefully provide better clarity on what the templates are doing.